ghost/stack.yml

```
@@ -18,9 +18,17 @@ services:
       url: http://localhost:8080
       # contrary to the default mentioned in the linked documentation, this image defaults to NODE_ENV=production (so development mode needs to be explicitly specified if desired)
       #NODE_ENV: development
+    volumes:
+     - some-ghost-data:/var/lib/ghost/content
```

Oh, it seems the `ymlfmt` test wants one extra space indent to keep things consistent.
```suggestion
      - some-ghost-data:/var/lib/ghost/content
```